robocopkaka/bootcamp-29-project
# #158264956 Fetch events when component mounts

#### What does this PR do?
fetch events when the Event component mounts
#### Description of Task to be completed?
* fetch events when the Event component mounts
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* enter the details of the center
* click on `Add Center`
* add at least ten more centers
* click on `Centers`
* select a center
* click on the red add button at the bottom right corner
* enter the details of the event
* click on `Add Event`
* you should see the newly created event on the center's page
* add some more events
* click on the 'Profile' dropdown then the `Profile` link
* you should see some events
* click on `Events` on the navbar
* you should see the first page of all the events on the system 
#### Any background context you want to provide?
the action to fetch events wasn't dispatched if you mounted the Event component after visiting a page that had events in it
#### What are the relevant pivotal tracker stories?(if applicable)
#158264956
